clickhouse/datadog_checks/clickhouse/queries.py

```diff
@@ -14,13 +14,23 @@
             'type': 'match',
             'source': 'value',
             'items': {
+                'BackgroundBufferFlushSchedulePoolTask': {
+                    'name': 'background_pool_buffer_flush.task.active',
+                    'type': 'gauge',
+                },
```

Comment:
```suggestion
                    'name': 'background_pool.buffer_flush_schedule.task.active',
                    'type': 'gauge',
                },
```

The other metrics similarly named seems to use `background_pool.` as a heirarchy